python/text-to-speech/main.py

```diff
@@ -0,0 +1,292 @@
+"""Synthesize text to speech using Google, Azure and AWS API."""
+# Standard library
+import abc
+import base64
+
+# Third party
+import boto3
+from google.cloud import texttospeech
+import requests
```

Comment:
```suggestion
import boto3
import requests
from google.cloud import texttospeech
```

Typically the `import` lines are sorted first followed by the `from` lines.

Reply:
Ohh that makes sense.